packages/amplication-git-service/src/contracts/GitResourceMeta.ts

```diff
@@ -1,3 +1,4 @@
+//TODO: REMOVE CLASS VALIDATOR!!!!!! ACCESS DEPENDENCY!!!
```

Comment:
Can you elaborate on this comment? Not sure if I understand 😄

Reply:
I added the `class-validator` dependency to the `git-service` library because it was missing. but the true story is that it shouldn't be there in the first place

Reply:
Can you refactor this comment to explain though? I couldn't understand what this means at first sight, and with open-source presence I think we should stick to more professional comments if we choose to use them.